src/Search_Replace_Command.php

```diff
@@ -108,7 +116,50 @@ class Search_Replace_Command extends WP_CLI_Command {
 	 * : Produce report. Defaults to true.
 	 *
 	 * [--report-changed-only]
-	 * : Report changed fields only. Defaults to false.
+	 * : Report changed fields only. Defaults to false, unless logging, when it defaults to true.
+	 *
+	 * [--log[=<file>]]
+	 * : Log the items changed. If <file> is not supplied or is "-", will output to STDOUT.
+	 * Warning: causes a significant slow down, similar or worse to enabling --precise or --regex.
+	 *
+	 * [--log_context=<before_num,after_num>]
+	 * : Number of characters to display before and after the match. One number sets both before and after. Two comma-separated numbers set before and after respectively. Defaults to 40. Ignored if not logging.
+	 *
+	 * [--log_prefixes=<old_prefix,new_prefix>]
+	 * : The old and new prefixes to prepend to the log lines for old match and new replacement, comma-separated. Defaults to '< ,> '. Use ',' for no prefixes. Ignored if not logging.
+	 *
+	 * [--log_colors=<table_column_id_color,old_color,new_color>]
+	 * : Percent color codes to use, comma-separated. The first is for 'table:column:id', default '%B'. The second is for the old match, default '%R'. The third is for the new replacement, default '%G'. The defaults are only used if logging to STDOUT. Use ',,' for no colors. Ignored if not logging.
+	 *
+	 * The percent color codes available are:
+	 * '%y' Yellow (dark) (mustard)
+	 * '%g' Green (dark)
+	 * '%b' Blue (dark)
+	 * '%r' Red (dark)
+	 * '%m' Magenta (dark)
+	 * '%c' Cyan (dark)
+	 * '%w' White (dark) (light gray)
+	 * '%k' Black
+	 * '%Y' Yellow (bright)
+	 * '%G' Green (bright)
+	 * '%B' Blue (bright)
+	 * '%R' Red (bright)
+	 * '%M' Magenta (bright)
+	 * '%C' Cyan (bright)
+	 * '%W' White
+	 * '%K' Black (bright) (dark gray)
+	 * '%3' Yellow background (dark) (mustard)
+	 * '%2' Green background (dark)
+	 * '%4' Blue background (dark)
+	 * '%1' Red background (dark)
+	 * '%5' Magenta background (dark)
+	 * '%6' Cyan background (dark)
+	 * '%7' White background (dark) (light gray)
+	 * '%0' Black background
+	 * '%8' Reverse
+	 * '%U' Underline
+	 * '%F' Blink (unlikely to work)
+	 * They can be concatenated.
```

Comment:
"They can be concatenated" needs an additional newline.

Reply:
This whole color stuff can now be just dropped.